pkg/loop/internal/pb/pipeline_runner.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb";
+
+package loop;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+
+message Options {
+  google.protobuf.Duration max_task_duration = 2;
```

Comment:
I am surprised there is special type for `Duration`, since it is just `int64` in Go. Is this for language interoperability?

Reply:
I think so: Go's duration is limited to ~290 years according to the docs, whereas since this proto models seconds and nanoseconds in different int fields (int64 and int32) respectively, it can achieve longer durations.